ci/check_style.sh

```diff
@@ -14,5 +14,12 @@ rapids-dependency-file-generator \
 rapids-mamba-retry env create --yes -f env.yaml -n checks
 conda activate checks
 
+# get config for cmake-format checks
```

Comment:
The diff from this change doesn't appear to be too large, so I am okay with including it in this PR.